package.json

```diff
@@ -7,7 +7,7 @@
     "test": "test"
   },
   "scripts": {
-    "prepublish": "npm run build",
+    "preinstall": "npm run build",
```

Comment:
My apologies, that shouldn't be changed. Please reject this pull request anyway as there is one test failing due to changes I did. I will send another pull request later. 

I was trying to make our project package.json depend on my forked repo and wasn't able to get it work because the way npm works. So, made this change and the change below. 

Your work on this project is really helpful. So thank you.